### PR TITLE
[Chat] Rename ImageGallery related Chat Theme

### PR DESCRIPTION
**Pull request by Leah-Xia-Microsoft**
# What

Rename ImageGallery related Chat Theme  base on the API review feedback

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/review/beta/react-components.api.md [hunk 1]
```diff
@@ -490,11 +490,11 @@ export interface ChatMessage extends MessageCommon {
...
     chatPalette: {
-        modalOverlayBlack: string;
-        modalTitleWhite: string;
-        modalButtonBackground: string;
-        modalButtonBackgroundHover: string;
-        modalButtonBackgroundActive: string;
+        imageGalleryOverlayBlack: string;
+        imageGalleryTitleWhite: string;
+        imageGalleryDefaultButtonBackground: string;
+        imageGalleryButtonBackgroundHover: string;
```
Comment (JamesBurnside) [inline packages/react-components/review/beta/react-components.api.md:496]:
```suggestion
        imageGalleryDefaultButtonBackgroundHover: string;
```

(Unless all buttons and future buttons (dismiss included) use the same background hover color)

File: packages/react-components/review/beta/react-components.api.md [hunk 2]
```diff
@@ -490,11 +490,11 @@ export interface ChatMessage extends MessageCommon {
...
+        imageGalleryButtonBackgroundActive: string;
```
Comment (JamesBurnside) [inline packages/react-components/review/beta/react-components.api.md:497]:
```suggestion
        imageGalleryDefaultButtonBackgroundActive: string;
```

(Unless all buttons and future buttons (dismiss included) use the same background active color)

File: packages/communication-react/review/stable/communication-react.api.md [hunk 3]
```diff
@@ -1748,13 +1788,15 @@ export const createAzureCommunicationCallAdapterFromClient: (callClient: Statefu
...
-export const createAzureCommunicationCallWithChatAdapterFromClients: ({ callClient, callAgent, callLocator, chatClient, chatThreadClient, callAdapterOptions }: AzureCommunicationCallWithChatAdapterFromClientArgs) => Promise<CallWithChatAdapter>;
+export const createAzureCommunicationCallWithChatAdapterFromClients: ({ callClient, callAgent, callLocator, chatClient, chatThreadClient, callAdapterOptions, chatAdapterOptions }: AzureCommunicationCallWithChatAdapterFromClientArgs) => Promise<CallWithChatAdapter>;
 
 // @public
 export const createAzureCommunicationChatAdapter: ({ endpoint: endpointUrl, userId, displayName, credential, threadId }: AzureCommunicationChatAdapterArgs) => Promise<ChatAdapter>;
 
 // @public
-export function createAzureCommunicationChatAdapterFromClient(chatClient: StatefulChatClient, chatThreadClient: ChatThreadClient): Promise<ChatAdapter>;
+export function createAzureCommunicationChatAdapterFromClient(chatClient: StatefulChatClient, chatThreadClient: ChatThreadClient, options?: {
+    credential?: CommunicationTokenCredential;
```
Comment (vhuseinova-msft) [inline packages/communication-react/review/stable/communication-react.api.md:1798]: Should  `AzureCommunicationChatAdapterOptions` type be used here?

File: packages/react-components/review/beta/react-components.api.md (hunk 1 above)
Comment (Leah-Xia-Microsoft) [reply packages/react-components/review/beta/react-components.api.md:496]: Currently the hover and active color are used by both the default download button and the close button

File: packages/react-components/review/beta/react-components.api.md (hunk 2 above)
Comment (Leah-Xia-Microsoft) [reply packages/react-components/review/beta/react-components.api.md:497]: Same here

**Review by JamesBurnside — APPROVED** [review]

**Review by vhuseinova-msft — APPROVED** [review]

**Pull request merged**